bin/modules-docs.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var https = require('https');
+var logger = require('console');
+
+var docs = path.join(__dirname, '..', 'docs', 'modules');
+var moduleReadmes = {
+    'geval': ['Raynos/geval', '2.1.1', 'README.md'],
```

Comment:
Yeah, nice idea. Or `npm.getCurrentPackageVersion('geval')`, or grab file from `node_modules`. I prefer the second. I'll make.